examples/v1beta1/github/github-eventlistener-interceptor.yaml

```diff
@@ -15,13 +15,17 @@ spec:
                 secretName: github-secret
                 secretKey: secretToken
             - name: "eventTypes"
-              value: ["pull_request"]
-        - name: "only when PRs are opened"
-          ref:
-            name: "cel"
-          params:
-            - name: "filter"
-              value: "body.action in ['opened', 'synchronize', 'reopened']"
+              value: ["pull_request", "issue_comment"]
+            - name: "ownersCheckEnabled"
+              value: true
+            - name: "githubToken"
+              value:
+                secretName: github-token
+                secretKey: secretToken
```

Comment:
instead of secretToken how about using personal_access_token/personalAccessToken ? in order to differentiate keys

Reply:
sure, I will change the name